### ISSUE TITLE
Forced switching to dedicated videocard

### ISSUE THREAD

**Issue by MrVerard** [AdguardTeam/AdGuardForSafari#110]
### Steps to reproduce

1. Normal using of application
2. Default settings and filters
3.
4.

### Actual behavior
MacOs switches to dedicated video card even on blank pages of safari, no other apps are running on system or safari

<details><summary>Screenshots:</summary>

</details>

### Your environment

* AdGuard for Safari version: 1.2.0
* Environment name and version: 12.0.2 (13606.3.4.1.4)
* Operating System: macOS High Sierra 10.13.6
* Filters you use in AdGuard: standard
* Any other browser extensions you have installed:none

**Comment by maciboy** [comment]
Hi,
this is indeed the only major issue I came across…
I'm also using a MacBook Pro with integrated and dedicated graphics.
Using gfxCardStatus one can see that AdGuard for Safari is triggering the dedicated card.
I assume this is due the the Electron app and due to the usage of Chromium within the bundle :(

Spotify earlier had the same issue, since their Mac app is also based on Chromium.
Meanwhile, they added a menu item to their app to let the user decide to turn on/off hardware acceleration.
Before that, one could run Spotify with an additional argument. I believe it was something like „—disable -gpu“…

So, long story short, I believe that for a adblocker nobody needs hardware acceleration.
Therefore, would it be possible to at least make it optional?
Here is a thread about a possible solution to turn off the usage of dedicated graphics:
https://github.com/electron-userland/electron-packager/issues/418

![bildschirmfoto 2018-12-22 um 15 25 21](https://user-images.githubusercontent.com/19333006/50375574-aedc8e00-05ff-11e9-9ba2-7668c37b76d5.png)

**Comment by ameshkov** [comment]
@maciboy thanks for the link, we'll take a look!

**Comment by maciboy** [comment]
Sounds good, thanks in advance!
Btw, this is a link to the Spotify community, where first the option to start Spotify with an additional flag is proposed and later the included menu item is shown:
https://community.spotify.com/t5/Desktop-Mac/Spotify-Desktop-Mac-requires-discrete-GPU/m-p/4476202#M532544

**Comment by maciboy** [comment]
Todays release notes say „[Fixed] Forced switching to dedicated video card“
However, for me Adguard is still triggering the Nvidia card…
Was that meant to be fixed with this update?

**Comment by MrVerard** [comment]
@maciboy I'll give it a try, mine mbp has an amd HD6xxx

**Comment by zzebrum** [comment]
@MrVerard @maciboy  how it goes? I can't repro the issue.

**Comment by zzebrum** [comment]
reopen if needed
